### PR TITLE
Add review

### LINKED ISSUE #1: 5 - Egen recension
På samma sida ska det vara möjligt att skicka in en egen recension, där man anger betyg samt sitt namn, och skriver en valfri kommentar. Formuläret ska skickas utan att sidan laddas om, d.v.s. med hjälp av webbläsarens fetch()

### PR DESCRIPTION
**Pull request by matodorovic**
### Description:
This PR allows a user to send a review for a movie and shows it between other reviews.

### Screenshot:
image

### Changes:

- Added review box 
- Sends review to api och renders recensions content without updating site
- Added post to CMS
- Addad change content of review box temporary after sending review

### Issue:
#1

### CODE REVIEW

**Comment by MrFabri** [comment]
Very nice!! Works perfectly

**Review by rfabricioflores — APPROVED** [review]
Very nice!! Works perfectly

**Review by OlliverLar — APPROVED** [review]

File: src/helpers/review.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+import fetch from "node-fetch";
+
+const API = "https://plankton-app-xhkom.ondigitalocean.app/api/reviews";
+
+export async function sendReview(movieId, data) {
+
+  const body = {
+    "data": {
```
Comment (richardolsson) [inline src/helpers/review.js:9]: This could just be `data` (not `"data"`) to be consistent with the following lines.

File: src/helpers/review.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      comment: data.comment,
+      rating: data.rating,
+      author: data.author,
+      verified: false,
+      movie: movieId,
+      createdAt: new Date().toISOString(),
```
Comment (richardolsson) [inline src/helpers/review.js:15]: Do you need to set this manually? I would expect that it's assigned by the CMS automatically.

File: static/js/movie.js [hunk 3]
```diff
@@ -226,3 +226,56 @@ function renderNextPage(data) {
...
+let reviewBox = document.querySelector(".addReview");
+
+// Change temporary content of adding review box into Thank you text 
+function changeContent() {
+  var form = document.getElementById("form")
+  var originalContent = reviewBox.innerHTML;
+  reviewBox.innerHTML = "Tack för din recension!";
+  setTimeout(function() {
+    reviewBox.innerHTML = originalContent;
+  }, 10000) };
```
Comment (richardolsson) [inline static/js/movie.js:243]: Nice feature!

The code formatting makes it a little bit difficult to read, however.

File: static/js/movie.js [hunk 4]
```diff
@@ -226,3 +226,56 @@ function renderNextPage(data) {
...
+
+  console.log(res.json())
+
+  document.querySelector("#rate").selectedIndex = 0;
+  document.querySelector("#addComment").value = "";
+  document.querySelector("#addName").value = "";
+
+  // Animation for the reviews
+  reviewContainer.style.height = `${reviewContainer.offsetHeight}px`;
+  reviewContainer.classList.add("screeningsAnim");
```
Comment (richardolsson) [inline static/js/movie.js:272]: I'm surprised to see `screeningsAnim` in the code for reviews. Is this reused code? Maybe abstract it by renaming it to something more generalized.

File: src/routes/api.routes.js [hunk 5]
```diff
@@ -10,6 +10,7 @@ router.get("/screenings", apiCtrl.getAllScreenings);
 router.get("/screenings/startpage", apiCtrl.startpageScreenings);
 router.get("/reviews", apiCtrl.reviews);
 router.get("/reviews/:id", apiCtrl.movieReviews);
+router.post("/reviews/:id", apiCtrl.review);
```
Comment (richardolsson) [inline src/routes/api.routes.js:13]: In my opinion, this path is not great. According to common practice, the ID following `/reviews` will be thought by most to be the ID of a review, but you're actually using it as the ID of the movie that this review relates to.

I would have picked another path for this, e.g. `/movies/:id/reviews`.

**Review by richardolsson — COMMENTED** [review]
Great work! This is working fine.

I did notice that the site is vulnerable to XSS however, so that's something to think about in the future. I've left some other feedback below, but no need to make any changes!

**Pull request merged**